src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -481,18 +481,29 @@ void ResumptionDataProcessor::DeleteGlobalProperties(
       application_manager_.ResetAllApplicationGlobalProperties(app_id);
 
   if (result.HasUIPropertiesReset()) {
-    smart_objects::SmartObjectSPtr ui_gl_props_reset_req =
+    smart_objects::SmartObjectSPtr msg_params =
         MessageHelper::CreateUIResetGlobalPropertiesRequest(result,
                                                             application);
-
-    ProcessHMIRequest(ui_gl_props_reset_req, false);
+    auto msg = MessageHelper::CreateMessageForHMI(
+        hmi_apis::messageType::request,
+        application_manager_.GetNextHMICorrelationID());
+    (*msg)[strings::params][strings::function_id] =
+        hmi_apis::FunctionID::UI_SetGlobalProperties;
+    (*msg)[strings::msg_params] = *msg_params;
+    ProcessHMIRequest(msg, false);
   }
   if (result.HasTTSPropertiesReset()) {
-    smart_objects::SmartObjectSPtr tts_gl_props_reset_req =
+    smart_objects::SmartObjectSPtr msg_params =
         MessageHelper::CreateUIResetGlobalPropertiesRequest(result,
```

Comment:
@LuxoftAKutsan Do we need the same`CreateUIResetGlobalPropertiesRequest`  for TTS and UI Properties? If yes may be do it once above the if's construction?

Reply:
@LuxoftAKutsan  it doesn't seem to be changed in this PR, but i think that under `result.HasTTSPropertiesReset()` condition, `MessageHelper::CreateTTSResetGlobalPropertiesRequest` should be called.

Reply:
@ZhdanovP @mked-luxoft  fixed